python/taichi/lang/kernel_impl.py

```diff
@@ -388,7 +398,7 @@ def lookup(self, args):
             )
 
         key = self.extract(args)
-        if key not in self.mapping:
+        if key not in self.mapping or self.mapping[key] is None:
```

Comment:
```suggestion
        if key not in self.mapping:
```
This change is unnecessary?